collect/cache/cache_test.go

```diff
@@ -160,11 +102,14 @@ func BenchmarkCache_Get(b *testing.B) {
 
 	c := NewInMemCache(b.N, metrics, &logger.NullLogger{})
 	populateCache(c, traces)
-	b.Run("InMemCache", func(b *testing.B) {
-		for traceID, _ := range traces {
-			c.Get(traceID)
-		}
-	})
+
+	// setup is expensive, so reset timer and report allocations
+	b.ResetTimer()
```

Comment:
I feel like these should be in the opposite order, although maybe it doesn't matter much.

Reply:
I flipped them but it doesn't seem has changed the result much